lib/absinthe/blueprint/input/object.ex

```diff
@@ -17,7 +17,7 @@ defmodule Absinthe.Blueprint.Input.Object do
   @type t :: %__MODULE__{
     fields: [Blueprint.Input.Field.t],
     flags: Blueprint.flags_t,
-    schema_node: nil | Absinthe.Type.InputObject.t,
+    schema_node: nil | Absinthe.Type.InputObject.t | Absinthe.Type.NonNull.t(Absinthe.Type.InputObject.t),
```

Comment:
I didn't even know you could do `@type t(x) :: %__MODULE__{of_type: x}`, good stuff.